index.html

```diff
@@ -213,10 +213,37 @@ <h2 class="h2-span">LET’S CONNECT</h2>
 					</ul>
 				</section>
 			</section>
+		</section>
 
-
+		<section class="form-container">
+			<h3 class="form-title">
+				Contact me
+			</h3>
+			<p class="form-p">
+				If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+			</p>
+			<form
+				class="form"
+				action="https://formspree.io/f/moqrapvz"
+				method="POST"
+			>
+				<ul class="form-ul">
+					<li class="input">
+						<input type="text" placeholder="username" name="user_name" maxlength="30" required>
+					</li>
+					<li class="input">
+						<input type="email" placeholder="email" name="email" required>
+					</li>
+					<li class="input">
+						<textarea name="message" placeholder="Write your message here" rows="8" maxlength="500" required></textarea>
+					</li>
+					<li class="input">
+						<button type="submit">Get in Touch</button>
+					</li>
```

Comment:
Great work implementing most of the requirements for this project 💪 👍. You just have one more project requirement to fulfill.

- [ ] Kindly implement the button interaction of `hover`, `active`, and `focus` for this button. You can find the color to use for the button interaction on this part of your Figma template on Figma's website;

![image](https://user-images.githubusercontent.com/77106962/144144958-7fc8917a-38bf-46bc-8739-28f634164a2d.png)

Reply:
HI @Gambit142, I will implement the changes as soon as possible.